android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -92,6 +91,8 @@ class RNIapModule(
             return
         }
 
+        billingClient = builder.setListener(this).build()
```

Comment:
Shouldn't we check this conditionally?
It'd be safer if we can set the listener only when the client is `closed`.

Reply:
Can we not consider as the client is closed when `billingClient.isReady == false`? We are checking this at the beginning of `initConnection`

Reply:
@Mkolakoglu Thanks for correcting the conversation. You're right 👍